libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayer.kt

```diff
@@ -108,7 +108,9 @@ internal class VoiceInstructionsTextPlayer(
             }
 
             override fun onError(utteranceId: String?) {
-                // Intentionally empty as deprecated
+                // Deprecated, may be called due to https://issuetracker.google.com/issues/138321382
```

Comment:
Wondering if `donePlaying` may be called twice and if that'd cause any issues 🤔

Reply:
That would mean Android platform calls both old and new version of onError in the same sequence, which is very unlikely. Additionally, looking at the implementation of donePlaying(), I see that it (normally) does nothing if called second time in the same sequence.